treeherder/services/management/commands/run_intermittents_commenter.py

```diff
@@ -0,0 +1,25 @@
+from django.core.management.base import BaseCommand
+
+from treeherder.services.intermittents_commenter.commenter import Commenter
+
+
+class Command(BaseCommand):
+    """Management command to manually initiate intermittent failures commenter
+        default is daily and non-test mode, use flags for weekly and test mode'"""
+
+    def add_arguments(self, parser):
+        parser.add_argument('-w', '--weekly', action='store_true', help='generates weekly, rather than daily, comment summaries')
+        parser.add_argument('-t', '--test', action='store_true', help='output comments to stdout rather than submitting to Bugzilla')
+
+    def handle(self, *args, **options):
+        weekly_mode = False
+        test_mode = False
+
+        if options['weekly']:
+            weekly_mode = True
+
+        if options['test']:
```

Comment:
If we go with this then I think it'd make sense to leave the --test flag as-is rather than renaming it to '--dry-run'. Then both of the add_args dests can be consistently named.